.circleci/config.yml

```diff
@@ -844,6 +873,42 @@ jobs:
             timeout 1800 bash -c 'while [[ "$(curl -n -s -o /dev/null -w ''%{http_code}'' https://api.mapbox.com/downloads/v2/mobile-maps-ios/releases/ios/$VERSION/MapboxMaps.zip)" != "200" ]]; do sleep 5; done' || false
           no_output_timeout: 30m
 
+  turf-integration:
+    <<: *base-job
```

Comment:
Can we drop the `base-job` injection here? I'd like to drop it someday from anywhere in CircleCI configuration if favor of shared executors.